shuup_tests/browser/admin/test_product_detail.py

```diff
@@ -56,8 +54,7 @@ def test_product_detail(browser, admin_user, live_server, settings):
 
     # Make sure that the tabs are clickable in small devices
     browser.driver.set_window_size(480, 960)
-    time.sleep(0.5)     # Otherwise other elements are still in the way, obscuring
-    click_element(browser, "#product-images-section", header_height=960)
+    click_element(browser, "#product-images-section", header_height=0)
```

Comment:
When the browser was scrolled all the way up with `header_height=960` and it tried to click this id that was outside the viewport, of course then the scroll-up arrow is blocking it:

>     selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <div id="product-images-section" class="content-block " style="display: none;">...</div> is not clickable at point (250, 807). Other element would receive the click: <i class="fa fa-chevron-up"></i>

So instead we scroll all the way down and then try to click the tab, since the "product images" tab is around the bottom of the list. This still makes sure that the tab itself is wide enough so that it can be clicked.